backend/router/userRouter.js

```diff
@@ -5,18 +5,28 @@ const checkUserSubscription = require("../middleware/ckeckUserSubscribtion.js");
 const {
```

Comment:
Update the auth middleware here as well in this file

Reply:
done